examples/rust_type_conversions/src/index.ts

```diff
@@ -237,60 +240,64 @@ type VariantWithInline = Variant<{
     inline_func: Func<Query<(param1: string) => string>>;
 }>;
 
-type FuncWithInline = Func<
-    Query<
-        (
-            inline_record: Record<{
-                one: boolean;
-                two: boolean;
-                three: boolean;
-            }>,
-            inline_variant: Variant<{ one: string; two: null; three: boolean }>,
-            inline_func: Func<Query<(param1: string) => string>>
-        ) => string
-    >
->;
+// TODO: Funcs with inline types don't currently work
```

Comment:
Do we have an issue for this?

Reply:
Just made one https://github.com/demergent-labs/azle/issues/1087